pdr_backend/ppss/sim_ss.py

```diff
@@ -94,11 +98,13 @@ def set_tradetype(self, tradetype: str):
 @enforce_types
 def sim_ss_test_dict(
     log_dir: str,
+    use_own_model: Optional[bool] = True,
     test_n: Optional[int] = None,
     tradetype: Optional[str] = None,
 ) -> dict:
     d = {
         "log_dir": log_dir,
+        "use_own_model": use_own_model,
```

Comment:
"use_own_model" needs to get added to unit tests, following the same pattern as other variables.